tutorials/jvm/migrate-to-kotlin/content.adoc

```diff
@@ -0,0 +1,127 @@
+= Migrate Java Enterprise Apps to Kotlin
+
+:type: text, image
+:topic: jvm
```

Comment:
The topic is already extracted from the folder strucutre

Reply:
And the topic 'jvm' does not make much sense. Why not just 'migration'?

Reply:
It's JVM languages, but migration also make sense. We will have a lot of migration tutorials.

So should I remove the `:topic:` line? I have it in pretty much all `content` files, copied from other files as well.

Reply:
@marcushellberg?

Reply:
Topic should only come from the folder structure. I may still have some incorrectly in my older articles. https://raw.githubusercontent.com/vaadin-learning-center/learning-content/development/TEMPLATE.adoc should be up to date with what we have agreed